gips/0001-gip-process.md

```diff
@@ -31,84 +31,137 @@ The GIP process includes both the stages and acceptance criteria that a proposal
 
-The GIPs repository is hosted on [Radicle](https://radicle.xyz/), a crypto-native decentralized alternative to Github. Currently, Radicle does not support an issue or Pull Request workflow, although this is being worked on.
+The GIPs repository is [hosted on GitHub](https://github.com/graphprotocol/graph-improvement-proposals).
+
+Proposals can be submitted by forking the repository and creating a Pull Request to include the proposal in Markdown format, following the GIP-0000 template.
+
+Discussions around proposals should be directed to the [GIPs and Governance](https://forum.thegraph.com/c/governance-gips/) category of The Graph's forum.
+
+**To avoid conflicts in GIP numbers, proposals should be posted as a Pull Request on GitHub with a GIP number _before_ being posted to the forum**.
+
+After creating the Pull Request, post a thread in the GIPs and Governance category with a link to the Pull Request. The post should have the same title as the GIP, and include the Abstract and Motivation sections but linking to GitHub for the full GIP text. (Older GIPs may include the full proposal text in the forum post, but this is now discouraged as it is hard to keep the forum and GitHub versions in sync).
+
+Editors will merge the PR into the main branch of the repo when it meets acceptance criteria, but the PR being merged is not a prerequisite for the community to discuss the GIP, or even for it to be implemented, accepted by the Council and deployed.
+
+After opening the Pull Request and posting on the forum, your proposal should be updated to have the URL of the forum thread in the "Discussions To" field of your proposal front matter. In this way, anyone viewing the proposal can find the conversation where a given proposal was discussed.
+
+## Choosing a GIP number
+
+Authors may attempt to self-assign the next available GIP number when submitting the Pull Request to the GitHub repo and before posting to the forum. When doing so, they should ensure:
+- That the GIP number is not used in an existing GIP in the repo, and
+- That the GIP number is not used in an open Pull Request in the repo.
+
+When creating the Pull Request, the title of the Pull Request should begin with the full GIP number (e.g. "GIP-0000"), to make it easier for other contributors to know that the number is taken.
+
+In case of conflicts, Editors may change the number for a GIP or ask the authors to change it, both in the repo and the forum post.
+
+If they prefer, authors may also ask the Editors to assign a number rather than self-assigning.
+
+# After submitting a proposal
+
+Publishing a GIP does not guarantee inclusion in the protocol. Protocol upgrades are implemented via decentralized governance, while the GIP Process is geared towards surfacing and building legitimacy for valuable protocol upgrades that can be considered in that process. Multiple GIPs may be bundled together into a release and be voted on as a single Graph Governance Proposal (GGP).
```

Comment:
This is the first time a GGP is mentioned. I think it's worth showing how such proposals are voted on and who usually creates them. A link to https://snapshot.org/#/council.graphprotocol.eth could help.

Reply:
update: you actually have more on this below under `# Getting a proposal accepted by the Council`. A reference to the snapshot page and the below section should suffice.